tests/encodehelp.h

```diff
@@ -152,13 +155,16 @@ static bool process_cmdline(int argc, char *argv[])
                     rcMode = string_to_rc_mode(optarg);
                     break;
                 case 3:
-                    ipbMode= atoi(optarg);
+                    ipbMode = atoi(optarg);
```

Comment:
OK, I see. I will recover it and keep it untouched.

Best Regards,
Xiaoxia Zhong

From: Xu Guangxin [mailto:notifications@github.com]
Sent: Monday, February 29, 2016 10:55 AM
To: 01org/libyami
Cc: Zhong, Xiaoxia
Subject: Re: [libyami] Add IDR interval setting to H.264 encoder (#447)


In tests/encodehelp.h<https://github.com/01org/libyami/pull/447#discussion_r54365051>:

> @@ -152,13 +155,16 @@ static bool process_cmdline(int argc, char *argv[])

>                      rcMode = string_to_rc_mode(optarg);

>                      break;

>                  case 3:

> -                    ipbMode= atoi(optarg);

> +                    ipbMode = atoi(optarg);

It's good but may introduce some side effect in this situation. Someone may relay on git blame find real author for this line. So modify it just for style may make them confused. So we'd better keep it untouched.
The format is not right because it's hard to check very details about code style. We find https://llvm.org/svn/llvm-project/cfe/trunk/tools/clang-format/git-clang-format is very helpful on this. Please use it to format all your patches.

command line is
git-format-clang --sytle=webkit HEAD^.
git commit -a --amend

thanks

—
Reply to this email directly or view it on GitHub<https://github.com/01org/libyami/pull/447/files#r54365051>.